test/dynamo/test_misc.py

```diff
@@ -3649,18 +3664,8 @@ def inner():
             result1, result2, _ = opt_fn()
             self.assertAlmostEqual(orig1 + 1 * i, result1)
             self.assertTrue(torch.allclose(orig2 + 10 * i, result2))
-            if i == 1:
```

Comment:
are those tests that you changed when you landed your thing? why do

Reply:
cause previously we wouldn't unspecialize floats on the first compile when dynamic=True